.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  # Only run push on main
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+  # Always run on PRs
+  pull_request:
+
+concurrency:
+  group: '${{ github.head_ref }}-${{ github.workflow }}'
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: ${{ matrix.ci_lint_version }} / JDK ${{ matrix.ci_java_version }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Generate cache key
+        run: ./.github/checksum.sh checksum.txt
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ matrix.job }}-${{ hashFiles('checksum.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-${{ matrix.job }}-
+
+      - name: Install JDK ${{ matrix.ci_java_version }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '19'
+
+      - name: Build
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: check
+
+      - name: (Fail-only) Bundle the build report
```

Comment:
Can remove this with the change below

Reply:
[7341b9c](https://github.com/slackhq/compose-lints/pull/4/commits/7341b9c6b04c1f8e96c8292f3574e3118cc7595c)